colcon_core/python_install_path.py

```diff
@@ -18,7 +18,9 @@ def get_python_install_path(name, vars_=()):
     """
     kwargs = {}
     kwargs['vars'] = dict(vars_)
+    # Avoid deb_system because it means using --install-layout deb
+    # The latter ignores --prefix and hardcodes it to /usr
```

Comment:
```suggestion
    # Avoid deb_system because it means using --install-layout deb
    # which ignores --prefix and hardcodes it to /usr
```

Initially I was commenting on the fact that I felt "the latter" ambiguously refers to `--install-layout deb` but I also wanted to propose an alternative. If you want to avoid a compound sentence over two lines then other alternatives are also reasonable.